modules/op-resolver-hub.adoc

```diff
@@ -0,0 +1,138 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-hub-specify_{context}"]
+= Specifying a remote pipeline or task using the Hub resolver
```

Comment:
standardize on caps, other places in the doc it's "hub resolver"

Reply:
standardized on not  using caps, this one fell through apparently